add.php

```diff
@@ -0,0 +1,55 @@
+<?php
+require_once 'src/ViewHelpers/CarsViewHelper.php';
+require_once 'src/Models/CarsModel.php';
+require_once 'src/Entities/Car.php';
+require_once 'src/Models/BodytypeModel.php';
+$db = new PDO('mysql:host=db; dbname=Cars', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$bodytype_model = new BodytypeModel($db);
+$bodytypes = $bodytype_model->getAllBodytypes();
+?>
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='index.css'>
+    <title>Add a Car</title>
+</head>
+<body>
+    <nav>
+        <a href='index.php'>Home</a>
+        <a href='add.php'>Add a Car</a>
+    </nav>
+
+<h1 class='header'>Add a Car</h1>
+   <form method="post" action='addHandling.php'>
+   <label for="model">Model:</label>
+   <input type="text" id="model" name="model" required>
+
+   <label for="make">Make:</label>
+   <input type="text" id="make" name="make" required>
+
+   <label for="bodytype">Body Type:</label>
+   <select id="bodytype" name="bodytype" required>
+    <?php
+    foreach($bodytypes as $bodytype)
+    {
+        echo "<option value={$bodytype->id}>{$bodytype->name}</option>";
+    }
```

Comment:
You should make a body type view helper with this - you'll probs want to display the same drop down again on the edit page. Don't forget a unit test for it :)